docs/kits/Business Partner Kit/Documentation BPDM/page_use-cases.md

```diff
@@ -0,0 +1,2294 @@
+                        }
+                    }
+                ],
+                "names": [],
+                "legalForm": null,
+                "status": null,
+                "profileClassifications": [],
+                "types": [],
+                "bankAccounts": [],
+                "roles": [],
+                "relations": [],
+                "currentness": "2022-10-14T15:00:06.325452Z"
+            }
+        },
+        {
+            "score": 0.0,
+            "legalEntity": {
+                "bpn": "BPNL00000006ME6Y",
+                "identifiers": [
+                    {
+                        "value": "63485ec63cd3b21131fb69ac",
+                        "type": {
+                            "technicalKey": "CDQID",
+                            "name": "CDQ Identifier",
+                            "url": ""
+                        },
+                        "issuingBody": {
+                            "technicalKey": "CDQ",
+                            "name": "CDQ AG",
+                            "url": ""
+                        },
+                        "status": {
+                            "technicalKey": "CDQ_IMPORTED",
+                            "name": "Imported from CDQ but not synchronized"
+                        }
+                    }
+                ],
+                "names": [],
+                "legalForm": null,
+                "status": null,
+                "profileClassifications": [],
+                "types": [],
+                "bankAccounts": [],
+                "roles": [],
+                "relations": [],
+                "currentness": "2022-10-14T15:00:06.325509Z"
+            }
+        },
+        {
+            "score": 0.0,
+            "legalEntity": {
+                "bpn": "BPNL00000006MF69",
+                "identifiers": [
+                    {
+                        "value": "63485ec63cd3b21131fb69ae",
+                        "type": {
+                            "technicalKey": "CDQID",
+                            "name": "CDQ Identifier",
+                            "url": ""
+                        },
+                        "issuingBody": {
+                            "technicalKey": "CDQ",
+                            "name": "CDQ AG",
+                            "url": ""
+                        },
+                        "status": {
+                            "technicalKey": "CDQ_IMPORTED",
+                            "name": "Imported from CDQ but not synchronized"
+                        }
+                    }
+                ],
+                "names": [],
+                "legalForm": null,
+                "status": null,
+                "profileClassifications": [],
+                "types": [],
+                "bankAccounts": [],
+                "roles": [],
+                "relations": [],
+                "currentness": "2022-10-14T15:00:06.325574Z"
+            }
+        }
+    ]
+}
```

Comment:
I think the other can get the idea with explicitely showing one or two legal entity data. Just cut the response short by providing "..." in the content list.